src/main/java/bio/terra/workspace/service/workspace/WorkspaceService.java

```diff
@@ -78,6 +79,10 @@ public UUID createWorkspace(WorkspaceRequest workspaceRequest, AuthenticatedUser
     createJob.addParameter(
         WorkspaceFlightMapKeys.WORKSPACE_STAGE, workspaceRequest.workspaceStage());
 
+    createJob.addParameter(
```

Comment:
Instead of passing each element of the request into the flight, we should pass the whole `workspaceRequest` as the request - instead of the `null` that is floating there in the `newJob()` call.

If there are serdes issues, they should be addressed in the WorkspaceRequest object. This simplifies request changes going forward. It can be done as a separate PR. If you choose that, can you make a ticket?

Reply:
Sure. I will file a follow up ticket so I can get this submitted.